synfig-studio/src/gui/docks/dock_toolbox.cpp

```diff
@@ -92,8 +92,8 @@ Dock_Toolbox::Dock_Toolbox():
 	Widget_Defaults* widget_defaults(manage(new Widget_Defaults()));
 
 	tool_box_paned = manage(new Gtk::Paned(Gtk::ORIENTATION_VERTICAL));
-	tool_box_paned->pack1(*scrolled_window, Gtk::PACK_EXPAND_WIDGET|Gtk::PACK_SHRINK, 3);
-	tool_box_paned->pack2(*widget_defaults, Gtk::PACK_EXPAND_WIDGET|Gtk::PACK_SHRINK, 3);
+	tool_box_paned->pack1(*scrolled_window, Gtk::PACK_EXPAND_WIDGET|Gtk::PACK_SHRINK, false);
+	tool_box_paned->pack2(*widget_defaults, Gtk::PACK_EXPAND_WIDGET|Gtk::PACK_SHRINK, false);
```

Comment:
I think it's wrong :(
The code before and after your changes ;)

Reply:
OMG now I just realized I only wrote half of may message :P Here it is:

Please check the method signature: https://developer.gnome.org/gtk3/stable/GtkPaned.html#gtk-paned-pack1

Reply:
@rodolforg 
It looks like he is right. This is the method signature:
`void pack1(Widget& child, bool resize, bool shrink);`

Reply:
The second argument is boolean, but he uses flags...

Reply:
@rodolforg 
This code was there before. Only the last flag has changed.

Reply:
It was wrong before XD